src/Microsoft.Health.Fhir.CosmosDb/Features/Storage/CosmosFhirDataStore.cs

```diff
@@ -364,6 +375,37 @@ internal async Task<(IReadOnlyList<T> results, string continuationToken)> Execut
             return (results, page.ContinuationToken);
         }
 
+        private void UpdateSortIndex(FhirCosmosResourceWrapper cosmosWrapper)
+        {
+            var searchParameters = _supportedSearchParameters.Value.GetSearchParameters(cosmosWrapper.ResourceTypeName)
```

Comment:
nit: Would be easier to read the code if we used the explicit type instead of var here.

Reply:
Lol, Resharper makes me forget this rule :P
![image](https://user-images.githubusercontent.com/197221/108887819-91305800-75bf-11eb-815a-e671e66ff3d2.png)